habitat/sims/habitat_simulator.py

```diff
@@ -420,6 +420,9 @@ def set_agent_state(
         state = self.get_agent_state(agent_id)
         state.position = position
         state.rotation = rotation
+
```

Comment:
Do we want to add comment here, to make sure it won't be removed mistakenly.

Reply:
Done